TACC/Core-CMS-Custom
# enhance(ecep): ecep_cms → ecep_assets

## Overview / Changes

Migrate ECEP from complex `ecep_cms` to simple `ecep_assets`.

## Related

- tested via https://github.com/TACC/Core-CMS/pull/856

## Testing

1. Open https://ecepalliance.org/.
2. Compare to https://pprd.ecep.tacc.utexas.edu/.
3. Verify no branding header.
4. Verify logo is unchanged **and** from remote URL.
5. Verify favicon is unchanged **and** from remote URL.

## UI

| | Before (Prod) | After (Test) |
| - | - | - |
| navigation | <img width="920" alt="ecep prod nav" src="https://github.com/user-attachments/assets/d14926d6-0ff1-49ef-ad12-7a4b87706467"> | <img width="920" alt="ecep test nav" src="https://github.com/user-attachments/assets/56f27c91-d529-4d81-803c-68e31a88ad23"> |
| news list | <img width="920" alt="ecep home news list" src="https://github.com/user-attachments/assets/928584d2-e89e-4f57-9374-699583b0ef76"> | <img width="920" alt="ecep test news list" src="https://github.com/user-attachments/assets/e273d5ba-a6e7-47c8-bdb6-ffdf54089e26"> |
| news page | <img width="920" alt="ecep home news page" src="https://github.com/user-attachments/assets/46f7eccc-30ef-480d-b422-d555e0333beb"> | <img width="920" alt="ecep test news page" src="https://github.com/user-attachments/assets/ec33c52c-b8ab-47f6-8e49-4a25279cb7ec"> |
| members | <img width="920" alt="ecep prod members" src="https://github.com/user-attachments/assets/1fdbcbca-fb59-4bf4-abdd-8e4efc0385fb"> |<img width="920" alt="ecep test members" src="https://github.com/user-attachments/assets/ed4e8a80-dd5e-4a2a-a467-af06c37a67b2"> |

<details><summary>Some pages have such different content, they are difficult to compare.</summary>

| | Before (Prod) | After (Test) |
| - | - | - |
| home page | ![ecep prod home](https://github.com/user-attachments/assets/7ba8e920-aec4-4f67-a6c6-1fca1541e39c) | ![ecep test home](https://github.com/user-attachments/assets/1ae0557e-d9ed-4b04-adf6-e4de89cf01f0) |

</detals>
